docker-compose.yml

```diff
@@ -1,11 +1,13 @@
 version: '3.8'
 
 services:
-  app:
+  service-app:
     build:
       context: ./service-app
     ports:
-      - "8081:8081"     
+      - "${HTTP_PORT}:${HTTP_PORT}"     
     environment:
-      - PORT=8081      
-    command: ./app
+      - SIRIUS_BASE_URL=${SIRIUS_BASE_URL}
+      - PORT=${HTTP_PORT}  
+      - PROJECT_PATH=${PROJECT_PATH}
```

Comment:
I think making these env vars is confusing, because it requires developers (and tools) to know appropriate values for them in order to run docker-compose. Particularly in the case of PROJECT_PATH and PORT, which should be fixed values (I think `/go` and `80` respectively).

Reply:
Sometimes I like to run tests in the IDE without docker running - so that was the reasoning for them, in those cases PROJECT_PATH is different, also HTTP_PORT can be conflicting sometimes. So it avoids replacing it everywhere each time.

Reply:
Note there is a .env.example file in SSM-23

Reply:
I don't _think_ that should conflict though: if `PROJECT_PATH=/go` is set in the docker-compose file, if you run `go test ./...` then it won't pay attention to anything in docker-compose and will carry on using your CLI/IDE env vars.

So I think hard-coding them in docker-compose does no harm, and makes it easier to run things through docker